packages/firestore/test/unit/local/local_store.test.ts

```diff
@@ -3098,4 +3098,57 @@ function indexedDbLocalStoreTests(
       .toReturnChanged('coll/a')
       .finish();
   });
+
```

Comment:
This test should move into local_store_indexeddb.test.ts. It's fine here for now though. I'd leave it here for now though, until I merge my PR that moves the other auto-indexing tests to local_store_indexeddb.test.ts.